src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -696,6 +727,14 @@ public Page Peek()
             return navigationPages[navigationPages.Count - 1];
         }
 
+        /// <summary>
+        /// Gets or sets if Navigator proceeds back navigation when back button or back key is pressed and released.
+        /// Back navigation pops the peek page if Navigator has more than one page.
+        /// If Navigator has only one page, then the current program is exited.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool EnableBackNavigation { get; set; } = true;
```

Comment:
shouldn't it be bindableProperty?

Reply:
Thank you for your comment!
You are right. I have updated it to be a bindable property.